.eslintcache

```diff
@@ -1 +1 @@
:"12"},{"size":2566,"mtime":1611627460655,"results":"15","hashOfConfig":"12"},{"size":650,"mtime":1611627460654,"results":"16","hashOfConfig":"12"},{"size":3862,"mtime":1611627460657,"results":"17","hashOfConfig":"12"},{"size":1329,"mtime":1611627460654,"results":"18","hashOfConfig":"12"},{"size":175,"mtime":1611627460657,"results":"19","hashOfConfig":"12"},{"size":1451,"mtime":1612156110985,"results":"20","hashOfConfig":"12"},{"size":436,"mtime":1611959641714,"results":"21","hashOfConfig":"12"},{"filePath":"22","messages":"23","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"sckwo3",{"filePath":"24","messages":"25","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"26","messages":"27","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"28","messages":"29","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"30","messages":"31","errorCount":0,"warningCount":1,"fixableErrorCount":0,"fixableWarningCount":0,"source":null},{"filePath":"32","messages":"33","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"34","messages":"35","errorCount":0,"warningCount":1,"fixableErrorCount":0,"fixableWarningCount":0,"source":null},{"filePath":"36","messages":"37","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"38","messages":"39","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"40","messages":"41","errorCount":0,"warningCount":1,"fixableErrorCount":0,"fixableWarningCount":0,"source":null},"/Users/comalvirdi/h4i2/rise-stand-strong/src/index.js",[],"/Users/comalvirdi/h4i2/rise-stand-strong/src/App.js",[],"/Users/comalvirdi/h4i2/rise-stand-strong/src/components/Main/Main.jsx",[],"/Users/comalvirdi/h4i2/rise-stand-strong/src/components/LoginPage/LoginPage.jsx",[],"/Users/comalvirdi/h4i2/rise-stand-strong/src/components/ForgotPassword/ForgotPassword.jsx",["42"],"/Users/comalvirdi/h4i2/rise-stand-strong/src/components/NewAccountPage/NewAccountPage.jsx",[],"/Users/comalvirdi/h4i2/rise-stand-strong/src/components/EditProfilePage/EditProfilePage.jsx",["43"],"/Users/comalvirdi/h4i2/rise-stand-strong/src/components/NavBar/NavBar.jsx",[],"/Users/comalvirdi/h4i2/rise-stand-strong/src/components/Dashboard/Dashboard.jsx",[],"/Users/comalvirdi/h4i2/rise-stand-strong/src/components/Dashboard/DemoCards.jsx",["44"],{"ruleId":"45","severity":1,"message":"46","line":18,"column":7,"nodeType":"47","messageId":"48","endLine":18,"endColumn":37},{"ruleId":"45","severity":1,"message":"46","line":29,"column":7,"nodeType":"47","messageId":"48","endLine":29,"endColumn":37},{"ruleId":"49","severity":1,"message":"50","line":13,"column":10,"nodeType":"51","messageId":"48","endLine":13,"endColumn":21},"no-alert","Unexpected alert.","CallExpression","unexpected","no-console","Unexpected console statement.","MemberExpression"]
```

Comment:
This is weird but I thought the .eslintcache was in the .gitignore? Just remove the file